src/librbd/mirror/DisableRequest.cc

```diff
@@ -84,6 +84,12 @@ Context *DisableRequest<I>::handle_get_mirror_image(int *result) {
 
 template <typename I>
 void DisableRequest<I>::send_get_tag_owner() {
```

Comment:
Might be nicer to abstract this into a `is_primary` check that either asks the journal or scans the snapshots